2014/12/24 18:45:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/testdrive/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/testdrive/index.php/
---
2014/12/24 18:46:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "outro".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('outro')
#1 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#2 /var/www/testdrive/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/testdrive/index.php/outro
---
2014/12/27 13:23:36 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /var/www/testdrive/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/testdrive/index.php/gii/model/index
HTTP_REFERER=http://localhost/testdrive/index.php/gii/model/index
---
2014/12/27 13:24:33 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[08006] [7] FATAL:  database "test_yii" does not exist' in /var/www/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /var/www/testdrive/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/testdrive/index.php/gii/model/index
HTTP_REFERER=http://localhost/testdrive/index.php/gii/model/index
---
2014/12/27 13:24:36 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[08006] [7] FATAL:  database "test_yii" does not exist' in /var/www/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /var/www/testdrive/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/testdrive/index.php/gii/model/index
HTTP_REFERER=http://localhost/testdrive/index.php/gii/model/index
---
2014/12/27 13:25:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /var/www/testdrive/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/testdrive/index.php/gii/model/index
HTTP_REFERER=http://localhost/testdrive/index.php/gii/model/index
---
2014/12/27 13:25:17 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /var/www/testdrive/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/testdrive/index.php/gii/model/index
HTTP_REFERER=http://localhost/testdrive/index.php/gii/model/index
---
